### PR TITLE
Add hash function docs

### PR DESCRIPTION
**Pull request by healthykim**
- Hash function
- Merkle tree

### CODE REVIEW

File: content/Pasted image 20240731133352.png
Comment (Insun35) [inline content/Pasted image 20240731133352.png:1]: Plz move this files from `content/` to `content/BasicCryptography`. Also move other files in same dir as well.

File: content/Basic Cryptography/images/Pasted image 20240730163028.png
Comment (Insun35) [inline content/Basic Cryptography/images/Pasted image 20240730163028.png:1]: Might be kinda irritating but could you match the names of all image files with their titles like `hash_function(1).png`, `hash_funtion(2).png`...? Would be better for maintaining.

**Review by Insun35 — COMMENTED** [review]
LGTM! Just a minor suggestion about img files.

File: content/Pasted image 20240731133352.png
Comment (healthykim) [reply content/Pasted image 20240731133352.png:1]: I moved those files to the images folder!

File: content/Basic Cryptography/images/Pasted image 20240730163028.png
Comment (healthykim) [reply content/Basic Cryptography/images/Pasted image 20240730163028.png:1]: Great! I renamed them :)

**Review by Insun35 — APPROVED** [review]

**Pull request merged**